tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -1021,6 +1021,39 @@ functionCall(
         ];
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFix74Cases
+     * @requires PHP 7.4
+     */
+    public function testFix74($expected, $input = null, array $config = [])
+    {
+        $this->fixer->configure($config);
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix74Cases()
+    {
+        return [
+            [
+                '<?php
+$fn = fn(
+    $test1,
+    $test2
+) => null;',
```

Comment:
unfortunately this case is failing. can you work on it, @julienfalque ?

Reply:
Fixed.

Reply:
big thanks!